hw_1_collections/hw_1_collections.py

```diff
@@ -0,0 +1,73 @@
+print('''Hi! I'm a python script that can work with a given RNA or DNA sequence. 
+For some reason I run in terminal and require manual input. Hope you will have at least some fun playing around with me)))0))0)
+
+Here's what I can do and understand:
+
+exit — завершение исполнения программы
+transcribe — напечатать транскрибированную последовательность
+reverse — напечатать перевёрнутую последовательность
+complement — напечатать комплементарную последовательность
+reverse complement — напечатать обратную комплементарную последовательность
+
+At least try to have some fun!
+''')
+
+nucl_comp_dct = {'RNA':{'A':'U', 'U':'A', 'G':'C', 'C':'G', 'a':'u', 'u':'a', 'g':'c', 'c':'g'}, 
+                 'DNA':{'A':'T', 'T':'A', 'G':'C', 'C':'G', 'a':'t', 't':'a', 'g':'c', 'c':'g'}}
+
+def is_seq_legit(seq):
+    if set(seq).issubset(nucl_comp_dct['RNA']) and not set(seq).issubset(nucl_comp_dct['DNA']):
+        return True, 'RNA'
```

Comment:
Мне больше нравится вот так, но это вкусовщина
```suggestion
    if set(seq) <= nucl_comp_dct['RNA'] and not set(seq) <= nucl_comp_dct['DNA']:
        return True, 'RNA'
```

Reply:
В целом, применение множеств здесь как нельзя кстати